runtime/oti/j9nonbuilder.h

```diff
@@ -2954,6 +2954,11 @@ typedef struct J9Class {
 #define J9INTERFACECLASS_ITABLEMETHODCOUNT(clazz) ((clazz)->totalInstanceSize)
 #define J9INTERFACECLASS_SET_ITABLEMETHODCOUNT(clazz, value) (clazz)->totalInstanceSize = (value)
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
This should probably be available always, not just with VT.

Reply:
The stride for normal arrays is 1 << arrayROMClass->instanceShape.

Reply:
Should I be adding another macro for normal arrays in that case?

Reply:
I don't think so - the existing macros are fine. Once you start setting the stride for VT arrays, you may as well set it for the other array types as well.